frontend/src/pages/detail/ModfiyBook.tsx

```diff
@@ -105,169 +111,205 @@ export default function BookDetail() {
         height: height,
         thickness: thickness,
         stock: 0,
+        urls: [""],
```

Comment:
Just out of curiosity, why is urls an array instead of just a string? Isn't there just one URL per book?

Reply:
Hosung thought maybe in the future we may need to support more than one image for a book such as a cover art, back cover, and side views. So we had this implemented just in case, but right now its just an array with one element.